### ISSUE TITLE
SUPPORT FOR ENV VAR GOOGLE_COMPUTE_REGION

### ISSUE THREAD

**Issue by kevinfaveri** [CircleCI-Public/gcp-gke-orb#9]
### Orb version

0.2.0

### What happened

I'm using a GKE cluster in a region which has three nodes. I can't publish with rollout-image since there is only a GOOGLE_COMPUTE_ZONE env var. If I try to put a region into GOOGLE_COMPUTE_ZONE env var Google Cloud throws an error: 
```
ResponseError: code=400, message='zone' field cannot be used to access GKE regional clusters. Use 'name' or 'parent' fields instead.
```

### Expected behavior

Should either work with GOOGLE_COMPUTE_ZONE or add a new GOOGLE_COMPUTE_REGION.

**Comment by dijitali** [comment]
I reckon this will actually just need a change to [CircleCI-Public/gcp-cli-orb ](https://github.com/CircleCI-Public/gcp-cli-orb/blob/master/src/commands/initialize.yml) which configures the gcloud CLI with the zone (or region).

The `update-kubeconfig-with-credentials` command should then work without any changes here.  Although other commands like `create-cluster` are going to need to support regional clusters too.

**Comment by llnformer** [comment]
the issue was fixed in the root orb, can you update this one too?

**Comment by xiaowei-routific** [comment]
We need the same thing. 🙏

**Comment by rogertsui0923** [comment]
Thanks for supporting regional clusters in version 1.2.0! One thing to watch out for is that if both zone and region are present, the orb will assume it is a zonal cluster, so if you are working with multiple clusters in the same CircleCI project, it might be a good idea to explicitly point the `google-compute-zone` or `google-compute-region` to different environment variables instead of relying on the default ones or use CircleCI contexts. I hope this saves someone somewhere a bit of time.

See https://github.com/CircleCI-Public/gcp-cli-orb/blob/master/src/commands/initialize.yml#L41-L44.

**Comment by EricRibeiro** [comment]
I reckon this change was merged on #34, so I'm closing this issue.
